var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -347,6 +389,7 @@ def cmake_args(self):
                 'ON' if '+shared' in spec else 'OFF'),
             '-DTrilinos_ENABLE_DEBUG:BOOL=%s' % (
                 'ON' if '+debug' in spec else 'OFF'),
+            '-DCMAKE_CXX_USE_RESPONSE_FILE_FOR_OBJECTS:BOOL=ON',
```

Comment:
What does this do?

Reply:
This shortens compile lines by placing some information into tmp files.